infrastructure/builder/src/generate/util.js

```diff
@@ -75,3 +75,8 @@ exports.gitLsFiles = async () => {
     .stdout.split(/\0/);
   return files;
 };
+
+/**
+ * Remove extension from `filename`
+ */
+exports.removeExtension = filename => filename.replace(/\.[^/.]+$/, '');
```

Comment:
is there a reason why regex is more preferable than methods from Node's `path` module?

Reply:
Good point. We could probably just use the `path` module. This was copy pasted after going through https://github.com/taskcluster/taskcluster/pull/216#discussion_r256600172.